src/data/contexts/alertsPromptContext.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { createContext, useContext, useState, ReactNode, useEffect } from 'react';
+import './alertsPromptContext.scss';
+import { CheckCircle, Info, Warning, XCircle } from '@phosphor-icons/react';
+
+const AlertsContext = createContext(undefined);
+
+export const useAlerts = () => {
+  const context = useContext(AlertsContext);
+  if (!context) {
+    throw new Error('useAlerts must be used within an AlertsPromptProvider');
+  }
+  return context;
+};
+
+const getAlertIcon = (severity) => {
+  switch (severity) {
+    case 'success':
+      return <CheckCircle />;
+    case 'error':
+      return <XCircle />;
+    case 'info':
+      return <Info />;
+    case 'warning':
+      return <Warning />;
+    default:
+      return <span>❔</span>;
+  }
+};
+
+export const AlertsPromptProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
The names of the file and exported members must align with the convention and with each other.
Rename the file to `AlertsContext` and the provider component to `AlertsProvider`.

Reply:
Also it doesn't seem like the provider is provided anywhere.
Please call the provider component in `main.tsx`.